metric/bigtable/src/main/java/com/spotify/heroic/analytics/bigtable/BigtableAnalyticsModule.java

```diff
@@ -143,10 +146,10 @@ public static Builder builder() {
 
     @NoArgsConstructor
     public static class Builder implements AnalyticsModule.Builder {
-        private Optional<String> project = Optional.empty();
-        private Optional<String> instance = Optional.empty();
-        private Optional<CredentialsBuilder> credentials = Optional.empty();
-        private Optional<Integer> maxPendingReports = Optional.empty();
+        private Optional<String> project = empty();
+        private Optional<String> instance = empty();
+        private Optional<CredentialsBuilder> credentials = empty();
+        private Optional<Integer> maxPendingReports = empty();
```

Comment:
Keep in mind that, these kind of refactoring tend to make things harder to merge when there are conflicts. Try to avoid them in the future, or do them in a distinct commit.